commondata_projections_L0/LEP3_240_H_HADR.yaml

```
@@ -0,0 +1,75 @@
+dataset_name: LEP3_240_H_HADR
+doi: https://doi.org/10.17181/3jjdh-6fz979
```

```suggestion
doi: https://doi.org/10.17181/3jjdh-6fz97
```
The last 9 in the DOI isn't needed.

---

Also, all the DOIs seem to refer to FCCee documents that say nothing about LEP3!